libr/asm/asm.c

```diff
@@ -189,12 +189,22 @@ static void plugin_free(RAsmPlugin *p) {
 	}
 }
 
-R_API RAsm *r_asm_new() {
+R_API RAsm *r_asm_new(RNum *num) {
```

Comment:
i wont touch that constructor. the r-arch api will affect all this and you can just instantaite num like we did all the time from core. it will be good to add refcounting to the RNum too

Reply:
ok 908b0e042e935b2324aa699c6229adf83d47b5bd